### DISCUSSION TITLE
[Question] Convert WorldGuard regions to GriefPrevention

### DISCUSSION THREAD

**Discussion by Xsm0deus** [GriefPrevention/GriefPrevention#515] [Ideas]
Hi on our server we use a protection plugin that uses worldguard. Regions have owners and members and I was wondering if it's possible to convert or not?

**Comment by RoboMWM** [comment]
If you mean converting worldguard regions to GriefPrevention claims, no there isn't in the base plugin, but there might be a tool/plugin that does this.

**Comment by TomLewis** [comment]
I think this would pair with:
#492 
#412

Once we can delete GP claims in a WE selection, it could work in reverse, do //rg sel regionname, that makes a WE selection and you can create a GP claim from that, OR selecting a massive area of WG regions to convert the load to GP regions.

**Comment by Xsm0deus** [comment]
I've started coding a plugin that converts it, but the issue is overlapping regions. Is there a way to add another area of land to an existing area. If they overlap?

**Comment by 112RG** [comment]
If they overlap then I would skip the region and log the fail so that you can manually fix it. GP doesn't allow overlapping claims so it may cause issues for player when resizing and such

**Comment by TomLewis** [comment]
if the overlap is fully contained treat it as a sub-claim.

**Comment by 112RG** [comment]
But then you have to deal with edge cases. From the sound of how regions are now its sounds like they overlap in corners and such

**Comment by TomLewis** [comment]
Thats why I said "fully contained".
